state/txindex/indexer.go

```diff
@@ -31,6 +31,12 @@ type TxIndexer interface {
 
 	//Set Logger
 	SetLogger(l log.Logger)
+
+	Prune(retainHeight int64) (int64, int64, error)
+
+	GetTxIndexerRetainHeight() (int64, error)
+
+	SetTxIndexerRetainHeight(retainHeight int64) error
```

Comment:
Similar to my comments on the `BlockIndexer` interface - can we not just call these functions `GetRetainHeight` and `SetRetainHeight`?

Reply:
Done